tests/e2e/testfiles/minimal-testfiles/storage_csm_powerscale_replica.yaml

```diff
@@ -28,4 +28,4 @@ spec:
           #  - name: "TARGET_CLUSTERS_IDS"
           #    value: "self"
         - name: dell-replication-controller-init
-          image: quay.io/dell/container-storage-modules/dell-replication-init:v1.0.0
+          image: dellemc/dell-replication-init:v1.0.0
```

Comment:
Is this expected to point to docker.io

Reply:
yes. dell-replication-init:v1.0.0 will not be released in quay. It exists in dockerhub